nncf/torch/dynamic_graph/io_handling.py

```diff
@@ -283,7 +287,7 @@ def from_nncf_config_dataloaders(cls, config: NNCFConfig) -> Optional["ExactInpu
                 wrapped_dataloader = wrap_dataloader_for_init(dataloader)
                 dataloader_output = next(iter(wrapped_dataloader))
                 args, kwargs = wrapped_dataloader.get_inputs(dataloader_output)
-                return ExactInputsInfo(args, kwargs)
+                return LoaderInputInfo(args, kwargs)
```

Comment:
`LoaderInputInfo` has no `__init__` right now. The `__init__` you want will be in the `ExactInputsInfo` class.

Reply:
done